Model/Cron.php

```diff
@@ -26,40 +26,41 @@ public function __construct(
 
     public function subscriberSync()
     {
-        if ($this->helperData->isCronEnabled()) {
-            $writer = new \Zend\Log\Writer\Stream(BP. '/var/log/smly_customer_cron.log');
-            $logger = new \Zend\Log\Logger();
-            $logger->addWriter($writer);
-            $logger->info('Running smaily customer synchronization!');
-            // Get last update time.
-            $last_update = $this->helperData->getLastCustomerSyncTime();
-            // Remove unsubscribers from Magento store.
-            $unsubscribers_list = $this->helperData->getUnsubscribersEmails(UNSUBSCRIBERS_BATCHES_LIMIT);
-            $this->customers->removeUnsubscribers($unsubscribers_list);
+        $writer = new \Zend\Log\Writer\Stream(BP. '/var/log/smly_customer_cron.log');
+        $logger = new \Zend\Log\Logger();
+        $logger->addWriter($writer);
+        $logger->info('Running smaily customer synchronization!');
+        // Get last update time.
+        $last_update = $this->helperData->getLastCustomerSyncTime();
 
-            // Import all customer to Smaily. List is in batches.
-            $subscribers_list = $this->customers->getList($last_update);
-            if (empty($subscribers_list)) {
-                $logger->info('No updated subscribers since last sync!');
+        // Remove unsubscribers from each Magento website separately.
+        foreach ($this->helperData->getWebsiteIds() as $websiteId) {
+            if ($this->helperData->isCronEnabledForWebsite($websiteId)) {
```

Comment:
Shouldn't this also check if extension is enabled?

Reply:
Yes, forgot enable checks. Added